### PR TITLE
508 for collections/download/index.php

### PR DESCRIPTION
**Pull request by NikitaSalikov**
# Changes Made:

- added Doctype and lang attribute
- added and fixed labels
- replaced fieldset with fieldset-like section

# Pull Request Checklist:

# Pre-Approval

- [X] There is a description section in the pull request that details what the proposed changes do. It can be very brief if need be, but it ought to exist.
- [ ] Hotfixes should be branched off of the `master` branch and **squash and merged** back into the `master` branch.
- [X] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [N/A] All new text is preferably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages)
- [N/A] There are no linter errors
- [X] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [X] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [N/A] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
- [X] Comment which GitHub issue(s), if any does this PR address
- [N/A] If this PR makes any changes that would require additional configuration of any Symbiota portals outside of the files tracked in this repository, make sure that those changes are detailed in [this document](https://docs.google.com/document/d/1T7xbXEf2bjjm-PMrlXpUBa69aTMAIROPXVqJqa2ow_I/edit?usp=sharing).

# Post-Approval

- [x] It is the code author's responsibility to merge their own pull request after it has been approved
- [x] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [ ] If this PR represents a hotfix into the `master` branch, a subsequent PR from `master` into `Development` should be made **merge** option (i.e., no squash).
- [ ] If the dev team has agreed that this PR represents the last PR going into the `Development` branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place. Follow the release protocol [here](https://github.com/BioKIC/Symbiota/blob/master/docs/release-protocol.md).
- [x] Don't forget to delete your feature branch upon merge. Ignore this step as required.

Thanks for contributing and keeping it clean!

### CODE REVIEW

File: collections/download/index.php [hunk 1]
```diff
@@ -100,7 +102,7 @@ function closePage(timeToClose){
 	}
 	?>
 	<div style="width:100%; background-color:white;">
-		<h2>Data Usage Guidelines</h2>
+		<h1>Data Usage Guidelines</h1>
```
Comment (Atticus29) [inline collections/download/index.php]: Internationalize.

File: collections/download/index.php [hunk 2]
```diff
@@ -132,7 +134,7 @@ function closePage(timeToClose){
 									such as substrate, associated collectors, verbatim description.
 								</div>
 								<input type="radio" name="schema" id="darwin-core" value="dwc" onclick="georefRadioClicked(this)" />
-								Darwin Core
+								<label for="darwin-core"> Darwin Core </label>
```
Comment (Atticus29) [inline collections/download/index.php]: Internationalize. Unless we discussed not doing it for this file (I can't remember).

File: collections/download/index.php [hunk 3]
```diff
@@ -159,23 +161,19 @@ function closePage(timeToClose){
...
-					</div>
-					<div class="sectionDiv">
-						<div class="labelDiv">
-							<label for="cset">Character Set:</label>
+							<input type="radio" name="format" id="csv-format" value="csv" CHECKED /><label for="csv-format"> Comma Delimited (CSV)</label><br/>
+							<input type="radio" name="format" id="tab-delimited-format" value="tab" /><label for="tab-delimited-format"> Tab Delimited</label><br/>
 						</div>
+					</fieldset>
+					<fieldset class="sectionDiv">
+						<legend> Character Set: </legend>
```
Comment (Atticus29) [inline collections/download/index.php]: Internationalize (maybe). See above.

File: collections/download/index.php [hunk 4]
```diff
@@ -187,16 +185,14 @@ function closePage(timeToClose){
 							<input type="radio" name="cset" id="utf-8" value="utf-8" <?php echo ($cSet=='utf-8'?'checked':''); ?> />
 							<label for="utf-8">UTF-8 (unicode)</label>
 						</div>
+				</fieldset>
+				<fieldset class="sectionDiv">
+					<legend> Compression: </legend>
```
Comment (Atticus29) [inline collections/download/index.php]: Internationalize (maybe). See above.

**Review by Atticus29 — COMMENTED** [review]
TVP has a few more recommendations:
[tvp_collections_download_index.pdf](https://github.com/BioKIC/Symbiota/files/13290984/tvp_collections_download_index.pdf)

**Review by Atticus29 — COMMENTED** [review]
Well done! Thank you for internationalizing all of that!
Hmm now I'm seeing a few TVP issues:
[tvp_collections_download_index.pdf](https://github.com/BioKIC/Symbiota/files/13382355/tvp_collections_download_index.pdf)

**Comment by NikitaSalikov** [comment]
> Well done! Thank you for internationalizing all of that! Hmm now I'm seeing a few TVP issues: [tvp_collections_download_index.pdf](https://github.com/BioKIC/Symbiota/files/13382355/tvp_collections_download_index.pdf)

It doesn't make sense... this looks like the first TVP report to me even though I made changes...

**Review by Atticus29 — COMMENTED** [review]
@NikitaSalikov , I'm still seeing TVP issues.

Do these new ones make more sense, or should I look more carefully?
[tvp_collections_download_index.pdf](https://github.com/BioKIC/Symbiota/files/13605671/tvp_collections_download_index.pdf)

**Review by Atticus29 — APPROVED** [review]
I made some final changes. TVP is happy except for a warning. Good to go.

**Pull request merged**